hack/quickstart/init-master.sh

```diff
@@ -7,8 +7,8 @@ CLUSTER_DIR=${CLUSTER_DIR:-cluster}
 IDENT=${IDENT:-${HOME}/.ssh/id_rsa}
 SSH_OPTS=${SSH_OPTS:-}
 
-BOOTKUBE_REPO=${BOOTKUBE_REPO:-quay.io/coreos/bootkube}
-BOOTKUBE_VERSION=${BOOTKUBE_VERSION:-v0.3.1}
+BOOTKUBE_REPO=${BOOTKUBE_REPO:-quay.io/quentin_m/bootkube}
+BOOTKUBE_VERSION=${BOOTKUBE_VERSION:-bootstrap_tls}
```

Comment:
Yeap, thanks. I didn't mean to include that and probably forgot it during the last rebase.